conf/ocsci/acm_2.8.yaml

```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  acm_version: "2.8"
```

Comment:
Also I think we should move those values also to OCP config file and the here you should add also channel I think. Because if you will have different version in ocp config then you will have mismatch.

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocp_version/ocp-4.13-config.yaml#L13C7-L13C31

Reply:
We already have in the OCP config but that represents GA values